src/vcpkg/registries.cpp

```diff
@@ -497,7 +497,7 @@ namespace
             return m_scfls.get_lazy(path, [&, this]() {
                 std::string spdx_location = "git+https://github.com/Microsoft/vcpkg#ports/";
                 spdx_location.append(port_name.data(), port_name.size());
-                return Paragraphs::try_load_port(m_fs, port_name, PortLocation{path, std::move(spdx_location)})
+                return Paragraphs::try_load_port_required(m_fs, port_name, PortLocation{path, std::move(spdx_location)})
```

Comment:
As an aside, I think this behavior is wrong, and this really should return the nullptr case. However, fixing that I believe is what resulted in https://github.com/microsoft/vcpkg-tool/pull/1486#issuecomment-2330811826

This PR intentionally does not change existing product behavior.